react-components/hello-world.md

```diff
@@ -0,0 +1,386 @@
+reportWebVitals();
+```
+
+<!-- Question 2 -->
+<!-- prettier-ignore-start -->
+### !challenge
+* type: checkbox
+* id: e447ec71
+* title: Hello, World!
+##### !question
+
+`src/index.js` imports several files. Select the files that are imported.
+
+##### !end-question
+##### !options
+
+* `React`
+* `ReactDOM`
+* `'./index.css'`
+* `App` from `'./App'`
+* `logo` from `'./logo.svg'`
+* `'./index.js'`
+
+##### !end-options
+##### !answer
+
+* `React`
+* `ReactDOM`
+* `'./index.css'`
+* `App` from `'./App'`
+
+##### !end-answer
+### !end-challenge
+<!-- prettier-ignore-end -->
+
+<!-- Question 3 -->
+<!-- prettier-ignore-start -->
+### !challenge
+* type: multiple-choice
+* id: 855add45
+* title: Hello, World!
+##### !question
+
+`ReactDOM.render( ... );` is a...
+
+##### !end-question
+##### !options
+
+* variable
+* function call
+* class
+
+##### !end-options
+##### !answer
+
+* function call
+
+##### !end-answer
+### !end-challenge
+<!-- prettier-ignore-end -->
+
+<!-- Question 4 -->
+<!-- prettier-ignore-start -->
+### !challenge
+* type: multiple-choice
+* id: b56db597
+* title: Hello, World!
+##### !question
+
+The following code snippet is in `src/index.js`.
+
+```js
+<React.StrictMode>
+  <App />
+</React.StrictMode>
+```
+
+This code is...
+
+##### !end-question
+##### !options
+
+* An import statement
+* A comment in React
+* The first argument when calling `ReactDOM.render()`
+* An object literal
+
+##### !end-options
+##### !answer
+
+* The first argument when calling `ReactDOM.render()`
+
+##### !end-answer
+### !end-challenge
+<!-- prettier-ignore-end -->
+
+`index.js` is responsible for being the starting point of our webapp.
+
+It is the entry point of our project; our app is configured to look for this file first when we run the app.
+
+This file renders our first _JSX_ (described in another lesson).
+
+```js
+<React.StrictMode>
+  <App />
+</React.StrictMode>
+```
+
+This code indicates that when we start our React webapp, it will _render_ an `App` component using `<App />`.
+
+In this curriculum, we will rarely touch `src/index.js`. Additionally, there are other lines of code in this file that is worth exploring with bigger projects.
+
+Instead, we'll focus on leveraging the `App` component.
+
+### Exploring `App.js`
+
+Consider the contents of `src/App.js`.
+
+```js
+import logo from "./logo.svg";
+import "./App.css";
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        <p>
+          Edit <code>src/App.js</code> and save to reload.
+        </p>
+        <a
+          className="App-link"
+          href="https://reactjs.org"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Learn React
+        </a>
+      </header>
+    </div>
+  );
+}
+
+export default App;
+```
+
+From reading `src/App.js`, answer the questions below.
```

Comment:
added a suggestion about forming hypotheses specifically because of one of the answers talking about the export statement, which to my knowledge isn't discussed until the components section.
```suggestion
From reading `src/App.js`, answer the questions below. If a question involves an expression that hasn't been discussed yet, we can hypothesize what it might do based on what we know about other languages, or from what a natural language reading might imply.
```